jerry-core/ecma/operations/ecma-bigint.c

```diff
@@ -76,56 +76,58 @@ ecma_bigint_parse_string (const lit_utf8_byte_t *string_p, /**< string represena
   ecma_bigint_digit_t radix = 10;
   uint32_t sign = (options & ECMA_BIGINT_PARSE_SET_NEGATIVE) ? ECMA_BIGINT_SIGN : 0;
 
-  if (size >= 3 && string_p[0] == LIT_CHAR_0)
+  const lit_utf8_byte_t *string_end_p = string_p + size;
+  const lit_utf8_byte_t *nonws_start_p = ecma_string_trim_front (string_p, string_p + size);
+  size = (lit_utf8_size_t) (string_end_p - nonws_start_p);
```

Comment:
You don't need `nonws_start_p`. Just simply write:
`string_p = ecma_string_trim_front (string_p, string_end_p);`